lib/MusicBrainz/Server/Data/Vote.pm

```diff
@@ -40,10 +40,10 @@ sub _column_mapping
 
 sub enter_votes
 {
-    my ($self, $editor, @votes) = @_;
+    my ($self, $editor, $votes, $override_privs) = @_;
```

Comment:
```suggestion
    my ($self, $editor, $votes, %opts) = @_;
```
It's a little more self-documenting if you accept `%opts` and then pass `override_privs => 1` at the call site.